hackathon_site/dashboard/frontend/src/components/cart/CartErrorBox/index.tsx

```diff
@@ -13,22 +13,11 @@ const CartErrorBox = () => {
             <Grid xs={12} sm={12} md={2} item />
             <Grid xs={12} sm={12} md={8} item>
                 {orderSubmissionError && cartQuantity > 0 && (
-                    <Alert severity="error" style={{ margin: "15px 0px" }}>
-                        {typeof orderSubmissionError === "object" ? (
-                            <>
-                                <AlertTitle>
-                                    Unable to submit your order because:
-                                </AlertTitle>
-                                <ul style={{ marginLeft: "20px" }}>
-                                    {orderSubmissionError?.map((error, index) => (
-                                        <li key={index}>{error}</li>
-                                    ))}
-                                </ul>
-                            </>
-                        ) : (
-                            orderSubmissionError
-                        )}
-                    </Alert>
+                    <AlertBox
+                        error={orderSubmissionError}
+                        type="info"
```

Comment:
good question! There are pre-defined types because this is an MUI component, if you do some digging into the source for for AlertBox, you'll see that the type is exactly as you guessed: `"error" | "info" | "success" | "warning"`
There will be an error if you pass in anything different because we are using typescript.